public/docs/ganache/using.md

```diff
@@ -0,0 +1,64 @@
+# Working with Ganache
+
+[Ganache](/ganache) is an application that generates a personal blockchain you can use to run tests, execute commands, deploy contracts, and inspect state.
+
+![Ganache logo](/docs/ganache/images/ganache-logo.png)
+
+It is available in both a graphical format and as a terminal application (which was previously known as the TestRPC). Ganache is available for Windows, macOS, and Linux.
```

Comment:
I'd call these a "desktop application" and "command line tool"

> It is available as both a desktop application as well as a command line tool (the command line tool was formerly known as the TestRPC).

Reply:
Also, `macOS` is the less common way to refer to a Mac, although probably technically correct. Can we revert the more colloquial "Mac"?

Reply:
I'll do a slight rephrase of the parenthetical to remove duplication, otherwise 👍 .